ports/atmel-samd/boards/metro_m0_express/pins.c

```diff
@@ -29,6 +29,5 @@ STATIC const mp_rom_map_elem_t board_global_dict_table[] = {
     { MP_ROM_QSTR(MP_QSTR_SCK),  MP_ROM_PTR(&pin_PB11) },
     { MP_ROM_QSTR(MP_QSTR_MOSI), MP_ROM_PTR(&pin_PB10) },
     { MP_ROM_QSTR(MP_QSTR_MISO), MP_ROM_PTR(&pin_PA12) },
-    { MP_ROM_QSTR(MP_QSTR_FLASH_CS),   MP_ROM_PTR(&pin_PA13) },
```

Comment:
I deleted it because it looked like something you might have added for debugging. There are no other named pins for the flash chip (e.g., no `FLASH_SCK`, etc.). And there were no similar named pins on the Feather M0, so I thought it was just for testing.

Reply:
Ok, that may be a hangover from when the SPI bus was shared. Should the others be added or this deleted?

Reply:
I'd leave it deleted for now because end-user use of these pins will mess up CIRCUITPY support. If we had some API for enabling/disabling CIRCUITPY on the flash chip, then someone could use the flash chip for something else besides a filesystem, but right now it's dedicated

Reply:
Sounds good. Approved!